src/Provider.tsx

```diff
@@ -17,7 +17,7 @@ export declare interface ProviderProps {
   client: ApolloClient;
 }
 
-export default class Provider extends Component<ProviderProps, any> {
+export default class ApolloProvider extends Component<ProviderProps, any> {
```

Comment:
Also, shouldn't `ApolloProvider` just use `composeApolloProvider` under the hood now?